jcl/src/jdk.management/share/classes/openj9/lang/management/internal/OpenJ9DiagnosticsMXBeanImpl.java

```diff
@@ -131,6 +131,23 @@ public String[] queryDumpOptions() {
 		}
 	}
 
+	/**
+	 * {@inheritDoc}
+	 */
+	@Override
+	public String getDumpOptions() {
+		String[] options = queryDumpOptions();
+		StringBuilder builder = new StringBuilder();
+		if (options.length > 0) {
+			builder.append(options[0]);
+		}
+		for (int i = 1; i < options.length; i++) {
+			builder.append(',');
+			builder.append(options[i]);
+		}
```

Comment:
This `for` loop can be within the `if` statement above, or the comma can be appended conditionally:
```java
		for (int i = 0; i < options.length; ++i) {
			if (i != 0) {
				builder.append(',');
			}
			builder.append(options[i]);
		}
```